### ISSUE TITLE
The maps on the Open Government page don't work with any assistive technology--they're not readable or navigable. Provide this content in another manner (we can discuss an accessible solution).

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#203]

**Comment by JeanneHolm** [comment]
This has been completed at https://www.data.gov/open-gov/
